psutil/_psbsd.py

```diff
@@ -455,6 +455,28 @@ def sensors_temperatures():
 
         return ret
 
+    def cpu_freq():
+        ret = []
+        num_cpus = cpu_count_logical()
+        for cpu in range(num_cpus):
+            try:
+                current, available = cext.cpu_frequency(cpu)
+                low = None
+                high = None
+                if available:
+                    try:
+                        low = int(available.split(" ")[-1].split("/")[0])
```

Comment:
Mmm. How does the string (`available`) look like in here?

Reply:
The `available` line looks like this:
`2701/45000 2700/45000 2600/42708 2500/40461 2400/38257 2300/36525 2200/34401 2100/32327 1900/28698 1800/26736 1700/24815 1600/23334 1500/21489 1400/19683 1300/18298 1200/16566`

It represents all the available levels of frequency the CPU can be in, not just min/max, that's why I thought it would be a better name